components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -274,7 +274,12 @@ public class ApplicationMgtDBQueries {
     public static final String REMOVE_UM_PERMISSIONS = "DELETE FROM UM_PERMISSION WHERE UM_ID = ?";
     public static final String REMOVE_UM_ROLE_PERMISSION = "DELETE FROM UM_ROLE_PERMISSION WHERE UM_PERMISSION_ID = ?";
 
-    // DELETE query - Oauth
+    // DELETE query - OAuth.
+    /**
+     * @deprecated Use the REMOVE_OAUTH_APPLICATION_WITH_TENANT query in
+     * {@link org.wso2.carbon.identity.application.mgt.dao.impl.ApplicationMgtDBQueries} class.
+     */
+    @Deprecated
     public static final String REMOVE_OAUTH_APPLICATION = "DELETE FROM IDN_OAUTH_CONSUMER_APPS WHERE CONSUMER_KEY=?";
```

Comment:
use the deprecated javadoc comment like below
    /**
     * @deprecated use {@link org.wso2.carbon.identity.application.mgt.dao.impl.ApplicationMgtDBQueries}
     */

Reply:
This is already addressed